Dockerfile

```diff
@@ -1,21 +1,21 @@
-FROM node:18
+FROM node:22
 
-COPY package-lock.json package.json  /app/
+# Create app directory

```

Comment:
Most of these changes are unnecessary and comments are redundant.

Reply:
I can drop the comments.

Reply:
The COPY of line 9 should be reverted too. I'd also suggest reverting the package lock changes.

Reply:
The build doesn't succeed without the dependency updates.